Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/Asset/AssetConnection.tsx

```diff
@@ -106,7 +104,25 @@ function AssetConnectionWindow(props: { Name: string, ID: number, TypeID: number
         if (index == -1 && localAssets.length > 0)
             setSelectedAssetID(localAssets[0].ID)
     }, [localAssets])
-   
+
+    React.useEffect(() => {
+        const handle = getAssetConnections();
+        return () => { if (handle != null && handle.abort != null) handle.abort(); }
+    }, [props.ID, trigger])
```

Comment:
why are we pulling `AssetConnections`? I thought this PR was about Locations?

Reply:
This was just a moved block of code, I'll fix it.